skills/src/lasr_skills/look_to_point.py

```diff
@@ -1,19 +1,66 @@
 import smach_ros
+import smach
+import actionlib
+import rospy
 from control_msgs.msg import PointHeadGoal, PointHeadAction
 from geometry_msgs.msg import Point, PointStamped
 from std_msgs.msg import Header
 
 
-class LookToPoint(smach_ros.SimpleActionState):
+# class LookToPoint(smach_ros.SimpleActionState):
+#     def __init__(self):
+#         super(LookToPoint, self).__init__(
+#             "/head_controller/point_head_action",
+#             PointHeadAction,
+#             goal_cb=lambda ud, _: PointHeadGoal(
+#                 pointing_frame="head_2_link",
+#                 pointing_axis=Point(1.0, 0.0, 0.0),
+#                 max_velocity=1.0,
+#                 target=PointStamped(
+#                     header=Header(frame_id="map"),
+#                     point=ud.point,
+#                 ),
+#             ),
+#             input_keys=["point"],
+#         )
+
+
+class LookToPoint(smach.State):
     def __init__(self):
-        super(LookToPoint, self).__init__(
-            "/head_controller/point_head_action",
-            PointHeadAction,
-            goal_cb=lambda ud, _: PointHeadGoal(
-                pointing_frame="head_2_link",
-                pointing_axis=Point(1.0, 0.0, 0.0),
-                max_velocity=1.0,
-                target=ud.pointstamped,
+        smach.State.__init__(
+            self,
+            outcomes=["succeeded", "aborted", "preempted", "timed_out"],
```

Comment:
```suggestion
            outcomes=["succeeded", "aborted", "timed_out"],
```
Since we never return "preempted"; note that you'll need to update the transitions in state machines that utilise this state.